x-pack/test/functional/apps/ml/embeddables/anomaly_embeddables_migration.ts

```diff
@@ -0,0 +1,121 @@
+        optionsJSON: '{"useMargins":true,"syncColors":false,"hidePanelTitles":false}',
+        timeRestore: true,
+        timeTo: '2016-02-11T00:00:00.000Z',
+        timeFrom: '2016-02-07T00:00:00.000Z',
+        refreshInterval: {
+          pause: true,
+          value: 0,
+        },
+        kibanaSavedObjectMeta: {
+          searchSourceJSON: '{"query":{"query":"","language":"kuery"},"filter":[]}',
+        },
+      },
+      coreMigrationVersion: '7.15.2',
+    },
+  },
+  {
+    type: ML_EMBEDDABLE_TYPES.ANOMALY_CHARTS,
+    panelTitle: 'ML anomaly charts',
+    dashboardSavedObject: {
+      attributes: {
+        title: `7.15.2 ML anomaly charts dashboard ${Date.now()}`,
+        description: '',
+        panelsJSON:
+          '[{"version":"7.15.2","type":"ml_anomaly_charts","gridData":{"x":0,"y":0,"w":38,"h":21,"i":"1155890b-c19c-4d98-8153-50e6434612f1"},"panelIndex":"1155890b-c19c-4d98-8153-50e6434612f1","embeddableConfig":{"jobIds":["fq_multi_1_ae"],"maxSeriesToPlot":6,"severityThreshold":0,"enhancements":{}},"title":"ML anomaly charts"}]',
+        optionsJSON: '{"useMargins":true,"syncColors":false,"hidePanelTitles":false}',
+        timeRestore: true,
+        timeTo: '2016-02-11T00:00:00.000Z',
+        timeFrom: '2016-02-07T00:00:00.000Z',
+        refreshInterval: {
+          pause: true,
+          value: 0,
+        },
+        kibanaSavedObjectMeta: {
+          searchSourceJSON: '{"query":{"query":"","language":"kuery"},"filter":[]}',
+        },
+      },
+      coreMigrationVersion: '7.15.2',
+    },
+  },
+];
+
+export default function ({ getService, getPageObjects }: FtrProviderContext) {
+  const esArchiver = getService('esArchiver');
+  const ml = getService('ml');
+  const PageObjects = getPageObjects(['common', 'timePicker', 'dashboard']);
+
+  describe('anomaly embeddables migration in Dashboard', function () {
+    this.tags(['mlqa']);
+
+    before(async () => {
+      await esArchiver.loadIfNeeded('x-pack/test/functional/es_archives/ml/farequote');
+      await ml.testResources.createIndexPatternIfNeeded('ft_farequote', '@timestamp');
+      await ml.testResources.setKibanaTimeZoneToUTC();
+      await ml.securityUI.loginAsMlPowerUser();
+
+      await ml.api.createAndRunAnomalyDetectionLookbackJob(JOB_CONFIG, DATAFEED_CONFIG);
+      await PageObjects.common.navigateToApp('dashboard');
+    });
+
+    after(async () => {
+      await ml.api.cleanMlIndices();
+    });
+
+    for (const testData of testDataList) {
+      const { dashboardSavedObject, panelTitle, type } = testData;
+      describe(`loads saved dashboard from version ${dashboardSavedObject.coreMigrationVersion}`, function () {
```

Comment:
This same description is used for the test below. I think it would be good to include the `panelTitle` in the description here too.

Reply:
Agreed - please let's not repeat this long title and have a shorter one for either the suite or the test.

Reply:
Updated here https://github.com/elastic/kibana/pull/116520/commits/7e51b015e16dd3ddfbae904fbb5778f529fa4f37